source/SIL.AppBuilder.Portal.Frontend/src/data/query.tsx

```diff
@@ -62,17 +63,25 @@ export function queryApi<T>(mapRecordsToProps, options?: IQueryOptions) {
   return InnerComponent => {
     class DataWrapper extends React.Component<T & WithDataProps, IState> {
       state = { result: {}, error: undefined, isLoading: false };
-
+      _isMounted: boolean = false;
```

Comment:
this needs to be _isMounted because react has an opinion about doing this and isMounted() is a deprecated react component method.... their opinion is cancellable promises... that's fine. they can have that opinion... for now... this works.